static/game/buttons.js

```diff
@@ -3,6 +3,7 @@ import { numSign } from "../modules/helpers.js";
 
 const noisy = false;
 const DEADZONE = 0.2;
+let USING_KEYBOARD = true;
```

Comment:
kind of minor, but doesn't THIS_CASE suggest it's something that never changes, since it's used a lot for pound defines in C and C++?

Reply:
Changed to `usingKeyboard`